Dockerfile.build

```
@@ -8,8 +8,10 @@ RUN CGO_ENABLED=0 go build -v -o /usr/local/bin/app ./cmd/...
 
 FROM alpine:3.19 AS certs
 RUN apk add --no-cache ca-certificates && update-ca-certificates
+RUN apk add --no-cache tzdata
```

```suggestion
RUN apk add --no-cache tzdata ca-certificates && update-ca-certificates
```

Just to reduce number of layers